lang/en.json

```diff
@@ -459,7 +459,8 @@
+    "yourLegalRightsAndResponsibilitiesContentHw": "<p class=\"govuk-body-l\">Before signing, you must read your legal rights and responsibilities.</p><h2 class=\"govuk-heading-m\">How your attorneys should act</h2><p class=\"govuk-body\">By signing your LPA, you are appointing your attorneys to make decisions for you.</p><p class=\"govuk-body\">Your attorneys must follow the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/mental-capacity-act-code-of-practice\">Mental Capacity Act Code of Practice</a>:</p><ul class=\"govuk-list govuk-list--bullet\"><li>They must assume you can make your own decisions, unless it is established that you cannot do so.</li><li>They must help you to make as many of your own decisions as you can.</li><li>They must take all practical steps to help you make a decision. They must only treat you as unable to make a decision if they have not succeeded in helping you make a decision through those steps.</li><li>They must not treat you as unable to make a decision because you have made an unwise decision.</li><li>They must act and make decisions in your best interests when you are unable to make a decision.</li><li>Before they make a decision or act for you, they must consider any option that is less restrictive of your rights and freedoms which might achieve the same outcome.</li></ul><h2 class=\"govuk-heading-m\">How your LPA can be used</h2><ul class=\"govuk-list govuk-list--bullet\"><li>When you sign your LPA, you are making a deed. This is a legal document which grants certain powers to your attorneys.</li><li>Your LPA can only be used once it is registered with the Office of the Public Guardian (OPG).</li><li>You can cancel your LPA at any time if you have mental capacity. <a class=\"govuk-link\" href=\"https://www.gov.uk/power-of-attorney/end\">Find out more about how to cancel your LPA</a>.</li><li>Your attorneys cannot use your LPA to make changes to your will.</li><li>Your LPA will expire when you die.</li></ul>",
```

Comment:
I'm half with you on keeping these as big chunks. But it did take me some time to figure out only the last set of bullets vary.